locale/en/blog/vulnerability/openssl-november-2022.md

```diff
@@ -1,7 +1,7 @@
 ---
 date: 2022-10-28T19:00:01.316Z
 category: vulnerability
-title: OpenSSL november security release
+title: OpenSSL November Security Release
```

Comment:
Probably should be `OpenSSL November security release` [per the Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/capitalization#sentence-style-capitalization-in-titles-and-headings), but also probably not worth "fixing" it. Just FYI for future reference.